RetailCoder.VBE/UI/RubberduckUI.sv.resx

```diff
@@ -1123,4 +1123,7 @@ Varning: Alla anpassade inställningar kommer att försvinna.    Din gamla fil k
   <data name="RenamePresenter_TargetIsInterfaceMemberImplementation">
     <value />
   </data>
-</root>
+  <data name="SourceControl_DeleteBranchButton" xml:space="preserve">
+    <value>Ta bort gren</value>
```

Comment:
@Vogel612 I would need to use Rubberduck myself to make sure it is correct with context considered. "Delete branch" does mean "Ta bort gren" though. So the translation looks correct to me.